src/olympia/files/tests/test_tasks.py

```diff
@@ -52,7 +52,7 @@ def test_repacking_xpi_files(self):
         # that structure is restored once the file has been moved.
         upload = self.get_upload('unicode-filenames.xpi')
         original_hash = upload.hash
-        repack_fileupload(upload.pk)
+        repack_fileupload(None, upload.pk)
```

Comment:
We could probably add a test case to make sure the task returns what we give to it.